base/acme/webapps/acme/help.jsp

```diff
@@ -0,0 +1,69 @@
+<!--
+Copyright Red Hat, Inc.
+
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+<h2>Account Management</h2>
+
+To create an ACME account:
+
+<pre>
+$ certbot register \
+    --server BASE_URL/directory \
+    -m &lt;email address&gt; \
+    --agree-tos
+</pre>
+
+To update an ACME account:
+
+<pre>
+$ certbot update_account \
+    --server BASE_URL/directory \
+    -m &lt;new email address&gt;
+</pre>
+
+To deactivate an ACME account:
+
+<pre>
+$ certbot unregister \
+    --server BASE_URL/directory
+</pre>
+
+<h2>Certificate Enrollment</h2>
+
+To request a certificate with automatic http-01 validation:
+
+<pre>
+$ certbot certonly --standalone \
+    --server BASE_URL/directory \
+    --preferred-challenges http \
+    -d server.example.com
+</pre>
+
+To request a certificate with manual dns-01 validation:
+
+<pre>
+$ certbot certonly --manual \
+    --server BASE_URL/directory \
+    --preferred-challenges dns \
+    -d server.example.com
+</pre>
+
+<h2>Certificate Revocation</h2>
+
+To revoke a certificate owned by the ACME account:
+
+<pre>
+$ certbot revoke \
+    --server BASE_URL/directory \
+    --cert-path /etc/letsencrypt/live/server.example.com/fullchain.pem
+</pre>
+
+To revoke a certificate associated with a private key:
+
+<pre>
+$ certbot revoke \
+    --server BASE_URL/directory \
+    --cert-path /etc/letsencrypt/live/server.example.com/fullchain.pem \
+    --key-path /etc/letsencrypt/live/server.example.com/privkey.pem
+</pre>
```

Comment:
We should probably add a link to the `certbot` docs for more advanced usages? 

(Also, does the `jsp` servlet know the address this is being requested over? Could we perhaps do `BASE_URL` substitution server-side rather than client-side? Hmmm...)

Reply:
I can add the link to `certbot` docs. Let me also look into server-side substitution.

Reply:
You don't have to :-) Just think it is a little weird to be doing JSP and then replacing BASE_URL on the client. :) Whatever works.

Reply:
I added the `certbot` link, but let me do the server-side substitution in another PR since that requires some investigation. I agree that the client-side substitution is weird.

Reply:
Sure, no problem.